src/app/boosts/layout.tsx

```diff
@@ -0,0 +1,13 @@
+import { DrawerProvider } from '@/context/DrawerContext';
+
+export default function BoostsLayout({
```

Comment:
Really don't like how nextJS requires this layout file which is pretty much copy and paste from other layout.